brianlong/www.validators.app
# 184324000 - add switcher for stake accounts

#### What's this PR do?
PR adds the switcher for stake accounts in terms of displaying the ones with less than 1 SOL.

#### How should this be manually tested?
- make sure to have valid stake accounts with validators included as records within the db (local)
- evaluate the appearance and place of the switcher button
- verify how does it act when switching in between options
- consider the following scenarios:
1) Validator includes multiple stake accounts with **less** than 1 SOL
2) Validator includes multiple stake accounts with **more** than 1 SOL
3) Validator includes mix stake accounts.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184324000)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
